indico/web/client/js/react/components/WTFDateField.jsx

```diff
@@ -45,6 +47,13 @@ export default function WTFDateField({
     dateField.value ? toMoment(dateField.value, moment.HTML5_FMT.DATE) : null
   );
   const clearRef = useRef(null);
+  const filter = (d, meta) => {
+    const dateStr = moment(d).format('YYYY-MM-DD');
```

Comment:
If `d` is a native date, we use this in some other places to format it to ISO format: `formatDate(ISO_FORMAT, d)` w/o first converting to a moment object

Reply:
Thanks pushed :)